includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -2920,9 +2920,20 @@ protected function prepare_payment_information_from_request( WC_Order $order ) {
 		}
 
 		// Use the dynamic + short statement descriptor if enabled and it's a card payment.
+		// For APMs, always use the full bank statement descriptor.
 		$is_short_statement_descriptor_enabled = 'yes' === $this->get_option( 'is_short_statement_descriptor_enabled', 'no' );
```

PHPMD (2.15.0)

[error] The variable $is_short_statement_descriptor_enabled is not named in camelCase. (undefined)

(CamelCaseVariableName)